### DISCUSSION TITLE
query: is there multiple PLMN support in 5G  ??

### DISCUSSION THREAD

**Discussion by matthewbalkwill** [srsran/srsRAN_Project#954] [General Help] [answered]
In 4G release notes (rel 22.10), I ee this support ..
- Allow to specify multiple PLMNs in SIB1

I am trying to trying to run a 5G version, but notice I can only set a single PLMN for the cell.
My config is 
    cu_cp:
      amf:
        ...
        supported_tracking_areas:
          - tac: 1
            plmn_list:
              - plmn: "99999"
                tai_slice_support_list:
                  - sst: 1
                    sd: 164
              - plmn: "00101"
                tai_slice_support_list:
                  - sst: 1
                    sd: 162
                  - sst: 1
                    sd: 163
    
    cell_cfg:
      ...
      plmn: "00101"
      tac: 1
      ...

I have some UE with test SIMs, with IMSIs starting "00101...." and some with IMSI starting "99999..."
When cell_cfg.plmn="00101", only the UEs with IMSIs starting "00101..." attempt to attach.
Similarly, when cell_cfg.plmn="99999", only the UEs with IMSIs starting "99999..." attempt to attach.
I guess this is to be expected.

Is there support for two PLMNs for 5G so that all my above test SIMs could attach.
If not, is it in the roadmap ?

**Comment by IsmaelBenitez** [comment]
Hello @matthewbalkwill , where you able to find a solution? I'm having the same problems as you. Thanks!

**Reply by matthewbalkwill** [reply]
No I do not believe there is a solution.
I had no feedback from anyone in srs so not sure if it is in the roadmap either.

**Comment by FabianEckermann** [comment, marked as answer]
Hi @matthewbalkwill @IsmaelBenitez,

It's possible to add additional PLMNs to the cell cfg:
```
cell_cfg:
  ...
  plmn: "00101"
  additional_plmns:
    - "99999"
  tac: 1
  ...
```
